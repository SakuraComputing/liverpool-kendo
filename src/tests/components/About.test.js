import React from 'react';
import AboutPage from '../../components/About';
import { shallow } from 'enzyme';

test(' AboutPage should render correctly', () => {
    const wrapper = shallow(<AboutPage />);
    expect(wrapper).toMatchSnapshot();
});