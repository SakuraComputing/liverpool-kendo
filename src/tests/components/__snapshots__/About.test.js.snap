// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` AboutPage should render correctly 1`] = `
<div>
  Kendo (剣道 kendō, lit. "sword way") is a modern Japanese martial art, which descended from swordsmanship (kenjutsu) and uses bamboo swords (shinai) and protective armour (bōgu). Today, it is widely practiced within Japan and many other nations across the world. Kendo is an activity that combines martial arts practices and values with strenuous sport-like physical activity.
</div>
`;
